report/src/introduction/waterfall_vs_agile.tex

```diff
@@ -1,2 +1,60 @@
+
+Choosing the appropriate model for a given project is important.
+Using a plan-driven approach is appropriate when you believe the entire process can be planned and laid out beforehand. That is, you expect the process to be deterministic.
+Agile teams, on the other hand, expects change and frequently makes sure they are working on what is creating the most value.
+
+\Citeauthor{boehmObservationsBalancingDiscipline} developed a method to select between plan-driven and agile approaches based on a risk-based analysis.
+This analysis is based on the \textit{size} of the team, \textit{criticality} of the project, as determined by the loss due to errors, \textit{dynamism} regarding how frequently requirements change, \textit{personnel} regarding the ability of the team, and \textit{culture}, regarding whether the team thrives on chaos or order. Depending on where the project places on these dimensions, agile, plan-drive, or a hybrid is appropriate.
+
+This project will be developed by a small team of 6 developers, which is far toward the agile end.
+Errors during the project would not have significant consequences, which also places the project in the agile end of the criticality dimension.
+During collaboration with the stakeholder, it is expected that requirements will be explored incrementally, and that they will provide feedback periodically. This places the project on the agile end of the dynamism dimension.
+Finally, the personnel \& culture dimensions also lead toward the agile end for this project.\cite{boehmObservationsBalancingDiscipline}
+
+Based on this analysis, it is concluded that an agile approach is an appropriate process model for this project.
+
+Agile is a set of values, and does not present practical activities which a process model would consist of. Therefore, the team has decided on Scrum, which is an agile framework that puts the values to practice. This decision was made due to prior experience with using the framework amongst team members.
+
+Scrum introduces a variety  of new practices and roles. In a Scrum team, there is a Scrum Master, a Product Owner, and developers. The Scrum Master is responsible for enforcing scrum and helping the team remain productive. Their goal is to guide the team towards continuous improvement. A Product Owner is responsible for maximizing value from the product. They are also accountable for managing the product backlog, which includes developing a product goal, creating backlog items and prioritizing them, and making sure the team understands this backlog.
+
+The process for working as a Scrum team boils down to repeatedly taking backlog items and working on them during a sprint. After the sprint, the team and stakeholders inspect the results and adjust for the next sprint. This process is what fosters continuous improvement by getting rapid feedback on work.
```

Comment:
Kilde paa at det koger ned til at repeatedly taking backlog items and working on them during a sprint.